recipes/clipcontext/meta.yaml

```diff
@@ -0,0 +1,34 @@
+package:
+  name: clipcontext
+  version: "0.1"
+
+source:
+  url: https://github.com/BackofenLab/CLIPcontext/archive/v0.1.tar.gz
+  sha256: df330517c397c8c3f1315fb020ff1dc611d923f9d1dcdb4de6fce3fda3c35c1b
+
+build:
+  number: 0
+  noarch: python
+  script: $PYTHON setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
can you please use pip install instead?

Reply:
Uuh yes .. is the current one outdated?

Reply:
Found this here:
  script: python -m pip install . --ignore-installed --no-deps -vv
Does this look okay?

Reply:
jepp